npdarrington/travel-tracker
# Nd main design refactor

### What’s this PR do?  
- Changed the way that the DOM was displaying trip cards at the bottom.
- Each Trip card now has a max and min width for wider larger screen and mobile views.
- Added new styles to reflect the HTML restructure.
- Changed the `displayAllTravelerTrips` method to display one at a time inside the trip-container section.
- On page load, displayAllTravelerTrips will first display upcoming trips.
- Added basic structure for the header at the top of the screen.
- Added favico based on logo icon.
 
### Where should the reviewer start?  
- Git pull this branch.
- Open index.js, domUpdates.js, index.html, _main.scss.
 
### How should this be manually tested?  
- Start the server with `npm start`.
- Open localhost:8080.
- Refresh page multiple times to ensure that the cards are displaying at the bottom correctly.
- Turn on mobile view, tablet view and desktop view along with responsive view to see how the form and trip-organizer section are on different screen sizes.
 
### Any background context you want to provide?  
- N/A
 
### What are the relevant tickets?  
- Closes #47 
- Closes #43 
- Closes #44 
- Closes #25 
 
### Screenshots (if appropriate)  
- N/A
 
### Questions: 
- N/A
